### DISCUSSION TITLE
Upgrade 3.0.0.19 to 3.0.9 - Checks writable files. No logs.

### DISCUSSION THREAD

**Discussion by nbenaissa** [nilsteampassnet/TeamPass#3864] [Q&A]
Hello, 

I'm looking for help/advice for this case:

I'm doing an upgrade from my teampass 3.0.0.19 to the version 3.0.9. 
After entering the credential to connect, it does a writing check for some files but it goes red for those files. I've tried a fresh install and i have the same error. 

The errors : 
"Something went wrong. Please check messages."

I found no logs about this error. 

![image](https://github.com/nilsteampassnet/TeamPass/assets/102679280/dcbf00a4-e475-414d-9bda-e1c4cabe3a3b)

I used this upgrade documentation
https://teampass.readthedocs.io/en/stable/install/upgrade/

OS : AlmaLinux9
Database version : MariaDB 10.5.16
PHP version : 8.1.14
Teampass version : 3.0.9

Thanks for your futur answer

**Comment by ismaelvillalba** [comment]
Try running the following command:

chmod a+rwx /(path to the files that gives you an error)

It will give read, write and execute permission to all users.

**Reply by nbenaissa** [reply]
Hello,

Nothing appended when I've changed it. Same error

But apparently the issue comes when you try to go to a version who is from 3.0.0.N to a version 3.0.N

I've tried to upgrade it version by version.
